google/ggrc-core
# GGRC-4240 Set default sorting in Bulk Update

# Issue description

Add default sorting for tasks in Bulk Update pop-up

# Steps to test the changes

1. Go to Tasks.
2. Open three dots menu, click Bulk Update.

**Expected result:** 'order_by' is sent in request body to sort by: Due Date, Task Assignee, Title.

# Solution description

Add order_by array to initial CycleTaskGroupObjectTask request

# Sanity checklist




# PR Review checklist
